04-GitRStudio.Rmd

```diff
@@ -105,41 +121,46 @@ Now that you have `staged` your changes, we are going to `commit` to the local r
 knitr::include_graphics("images/Git18.PNG")
 ```
 
-Every `commit` must be accompanied by a summary. It can be anything, but it's helpful to make it short, clear, and descriptive so you know what changes are represented in that commit. Regardless of method used to interact with git --- GitHub Desktop, RStudio, command line --- you must always provide a summary when committing.
+Every `commit` must be accompanied by a summary. It can be anything, but it's helpful to make it short, clear, and descriptive so you know what changes are represented in that commit. 
+Regardless of the method used to interact with Git --- GitHub Desktop, RStudio, command line --- you must *always* provide a summary when committing.
 
 Here are a few examples to help you get started:
 
--   first commit of data analysis code
+-   "first commit of data analysis code"
 
--   major updates to analysis. GLM to GAM
+-   "major updates to analysis. GLM to GAM"
 
--   changes to trends plots
+-   "changes to trends plots"
 
--   fix broken web links
+-   "fix broken web links"
 
-When you `commit` changes, .git saves every detail about the current state of the folder at that moment.
+When you `commit` changes, Git saves every detail about the current state of the folder at that moment.
 
-This is one of the most confusing things about git when you are just beginning because the word `commit` is both a verb and a noun. Once you commit (verb) the current state of your .git repo/folder it's referred to as a commit (noun).
+This is one of the most confusing things about Git when you are just beginning because the word `commit` is both a verb and a noun. Once you commit (verb) the current state of your Git repo/folder it's referred to as a commit (noun).
 
 Really if we were being grammatically correct, we would probably refer to it as a commitment, or something like that. But as it stands, the word `commit` is used for both the process and the state.
 
 ```{r image22, echo=FALSE, out.width="700px", fig.align='center'}
 knitr::include_graphics("images/Git19.PNG")
 ```
 
-> Note: all .git repos are initialized first in GitHub, the cloud version is the primary copy, referred to as the `origin`.
```

Comment:
Again this isn't completely true as repos can be initialized anywhere and out of order... I also found this note a bit out of place as I assumed it would refer back to the previous section. See what you think of my suggestion.